src/topology.jl

```diff
@@ -91,9 +171,17 @@ function Cart_shift(comm::Comm, direction::Integer, disp::Integer)
     Int(rank_source[]), Int(rank_dest[])
 end
 
-function Cart_sub(comm::Comm, remain_dims)
+"""
+    comm_sub = Cart_sub(comm::Comm, remain_dims)
+
+Create lower-dimensional Cartesian communicator from existent Cartesian
+topology.
```

Comment:
Maybe explain what `remain_dims` is (i.e. a boolean vector)

Reply:
Done in https://github.com/JuliaParallel/MPI.jl/pull/340/commits/c0a54f988cfc2213d876a2c2345e0e6eab464d9a